packages/utils/helpers/jasmine-13-helper.ts

```diff
@@ -68,12 +68,40 @@ jasmine.browserSupportsProtoAssignment = { __proto__: [] } instanceof Array;
 
 
 jasmine.ieVersion = ieVersion;
+
+jasmine.setNodeText = function(node, text:string) {
+    'textContent' in node ? node.textContent = text : node.innerText = text;
+};
+
+
+function cleanedHtml(node) {
+    var cleanedHtml = node.innerHTML.toLowerCase().replace(/\r\n/g, "");
+    // IE < 9 strips whitespace immediately following comment nodes. Normalize by doing the same on all browsers.
```

Comment:
i think we can also weed out < ie9 logic (not needed now, just noting)

Reply:
What do you think about IE11? IE7 to IE10 can be weed out. IE11-cases are rather few anyway..